en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the unified release policy for Silverstripe CMS and its commercially supported modules. Its purpose is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+## Release types and cadence
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its release philosophy.
+
+### Major releases `(x.0.0)`
+
+Major releases are the cornerstone for introducing significant evolution and breaking changes to the Silverstripe CMS platform.
```

Comment:
Where did this language come from? This is meant to be a blending of the two policies - I'd expect most if not all of the wording to come from them.

Reply:
Removed